app/views/users/mailer/notification.html.erb

```diff
@@ -1,13 +1,16 @@
-<p>Hello <%= @user.name %>!</p>
+<p>Hello <%= @user.name %>,</p>
 
-<p><%= I18n.t("notifications.email_title") %></p>
-
-<p>Type: <%= I18n.t("notifications.types.#{@notification.type_of}") %></p>
+<% unless @notification.deliver? %>
+  <p><%= I18n.t("notifications.email_title") %></p>
+  <p>Type: <%= I18n.t("notifications.types.#{@notification.type_of}") %></p>
+<% end %>
 
 <p>
-  <% if @notification.system_message? || @notification.deliver?   %>
+  <% if @notification.system_message? %>
     <% # We assume the system notification is clean %>
     <%= @notification.title.html_safe %>
+  <% elsif @notification.deliver? %>
+    <%= I18n.t("notifications.deliver.email_body").html_safe %>
```

Comment:
Can't you just put the locale text inside `@notification.title`, instead of doing this conditional here?